changelogs/fragments/398-docker-api.yml

```
@@ -0,0 +1,5 @@
+major_changes:
+  - "The collection now contains vendored code from the Docker SDK for Python to talk to the Docker daemon.
+     Modules and plugins using this code no longer need the Docker SDK for Python installed on the machine
+     the module resp. plugin is running on
```

```suggestion
     the module or plugin is running on
```
nit: maybe this is just my brain after a long day but I don't understand the "resp." conjunction 😖